build.sbt

```diff
@@ -65,3 +52,38 @@ ReleaseKeys.releaseProcess := Seq[ReleaseStep](
   pushChanges
 )
 
+lazy val root = (project in file(".")).
+  aggregate(faciaJson, fapiClient)
+
+
+lazy val faciaJson = project.in(file("facia-json"))
+  .settings(
+    crossScalaVersions := Seq("2.10.4", "2.11.4"),
+    scalaVersion := "2.11.4",
+    organization := "com.gu",
+    name := "facia-json",
+    libraryDependencies ++= Seq(
+      awsSdk,
+      commonsIo,
+      specs2,
+      playJson
+    )
+  )
+
+lazy val fapiClient = project.in(file("fapi-client"))
+  .settings(
+    scalaVersion := "2.11.4",
+    crossScalaVersions := Seq("2.10.4", "2.11.4"),
+    organization := "com.gu",
+    name := "fapi-client",
+    resolvers ++= Seq(
+      Resolver.file("Local", file( Path.userHome.absolutePath + "/.ivy2/local"))(Resolver.ivyStylePatterns),
+      "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"
+    ),
+    libraryDependencies ++= Seq(
+      "com.gu" %% "content-api-client" % "4.0-SNAPSHOT",
```

Comment:
Should probably be consistent with how dependencies are defined above (vals from a separate Dependencies config).

@rtyley showed me this originally, but it's a really good way of easily seeing all the dependencies of a project, but also making sure versions don't become out of sync between subprojects